app/models/host_status/execution_status.rb

```diff
@@ -1,17 +1,21 @@
 class HostStatus::ExecutionStatus < HostStatus::Status
+  # execution finished successfully
   OK = 0
```

Comment:
Ah, I haven't realized it's coming from the Foreman host status… nm then. Thanks for explanation